components/BlogFeed.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable react/no-danger */
+
+import React from 'react';
+import Truncate from 'react-truncate';
+import styled from 'styled-components';
+import MarkdownIt from 'markdown-it';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import CloudinaryImage from './CloudinaryImage';
+import Parser from './Parser';
+import theme from '../theme';
+
+export default function BlogFeed({ posts }) {
+  const md = new MarkdownIt();
+  if (posts === 'undefined') return null;
+
+  return (
+    <StyledBlogFeed>
+      <div className="blog blog__container">
+        <h2 className="blog__heading">TJI Blog</h2>
+        <span className="blog__tagline">
+          Where the TJI team posts blogs about our data, publications, analyses, and code.
+        </span>
+        <hr />
+        {!posts && <div>No posts!</div>}
+        <ul className="blog__posts">
+          {posts &&
+            posts.map(post => (
+              <li className="blog__post" key={post.slug}>
+                {post.attributes.hero && (
+                  <div className="blog__post__image">
+                    <CloudinaryImage
+                      url={post.attributes.hero}
+                      alt={post.attributes.title}
+                      maxWidth={theme.newsItemImageWidthPixels}
+                    />
+                  </div>
+                )}
+                <div className="blog__post__content">
+                  <h3>
+                    <Link href={{ pathname: `/post/${post.slug}` }}>
+                      <a className="blog__post__read-more" target="_blank" rel="noopener noreferrer">
+                        {post.attributes.title}
+                      </a>
+                    </Link>
+                  </h3>
+                  <div className="blog__post__details">
+                    <span className="blog__post__date">{moment(post.attributes.date).format('MMMM D, YYYY')}</span>
+                    <div className="blog__post__authors">
+                      By:&thinsp;
+                      {post.attributes.authors.length > 1
+                        ? post.attributes.authors.map(author => <span key={author}>{md.renderInline(author)}</span>)
```

Comment:
Do you think it would be worth getting a little fancier for multiple authors? I'm thinking something like...

2 authors: "Margarita Bronshteyn and Eva Ruth Moravec"
3 or more authors: "Margarita Bronshteyn, Eva Ruth Moravec, and Jen Udan"